Makefile.defs

```diff
@@ -32,10 +32,9 @@ CMAKE=cmake
 
 # set this to your ARM cross compiler:
 # set CC CXX AR LD RANLIB
-ifneq (,$(filter $(TARGET), kindle arm-generic))
-    export KINDLE=1
+ifneq (,$(filter $(TARGET), arm-generic))
```

Comment:
Should probably drop the brain-twisty filter trick for getting a logical OR since there's now only one test ;).

```ifeq ($(TARGET), arm-generic)```

(And yeah, that syntax trick kinda makes my brain hurt, but I think you got it right in your latest commit ;)).